permits/models.py

```diff
@@ -110,6 +111,11 @@ def __str__(self):
         return str(self.group)
 
 
+class PermitAdministrativeEntityQuerySet(models.QuerySet):
+    def filter_by_tags(self, tags):
+        return self.filter(tags__name__in=tags)
```

Comment:
Je ferais un `tags__name__in=[tag.lower() for tag in tags]`, pour éviter que le filtre `Yverdon` par exemple renvoie toutes les entités plutôt que celles qui ont le tag `yverdon`. Tu devrais aussi ajouter un `.distinct()` comme indiqué dans la doc (https://django-taggit.readthedocs.io/en/latest/api.html#filtering).

Reply:
Done